QUEST.tsv

```diff
@@ -122,82 +122,82 @@ xQUEST_20150317_000121	This mission makes me feel like I've seen enough monsters
 xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and upper streams.
 xQUEST_20150317_000123	Klaipeda Guard Captain
 xQUEST_20150317_000124	You're the one who…
-xQUEST_20150317_000125	It looks like you can see into the upper forest from here. {nl}Try not to provoke anything bigger than you.
-xQUEST_20150317_000126	You don't have the required quest item. Are you sure you want to continue?
+xQUEST_20150317_000125	It looks like you can see into the upper forest from here. {nl}Try not to get into trouble.
+xQUEST_20150317_000126	You don't have the required quest item. Are you sure you want to go on?
 xQUEST_20150317_000127	Klaipeda Girl
 xQUEST_20150317_000128	The man who lives next door to me used to be a monster hunter. {nl}He's retired now, though. It was too dangerous.
 xQUEST_20150317_000129	I think it would be nice if all these Revelators were a sign that the goddesses are back.
-xQUEST_20150317_000130	It might be silly to rely on hope, but it's better than being hopeless.
+xQUEST_20150317_000130	It might be silly to hope for the Goddess again, but it's better than being hopeless.
```

Comment:
Try "keep hoping for the Goddess", there are a lot of characters hoping for the return of the goddesses in this world.